prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh

```diff
@@ -32,7 +32,7 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
-CLUSTER_VERSION_PARAM="--cluster-version=1.10"
+CLUSTER_VERSION_PARAM="--cluster-version=1.11.5"
```

Comment:
Please do not set the version in a `x.y.z` format as the last parameter is very often changed and deprecated in gcloud, which causes problem like this:

`ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=No valid versions with the prefix "1.10.7" found.`

Please set it to `1.11` or add a gcloud lookup to get the last valid `1.11.x` version

Reply:
Applied the feedback. Changed to use `1.11`